### DISCUSSION TITLE
Where are the default project files saved?

### DISCUSSION THREAD

**Discussion by retcons** [phcode-dev/phoenix#736] [Q&A] [answered]
I made a project using the default project files. I want to export/download my project but I don't know where the files are located. On Opera GX, OS is Windows 10.

**Comment by abose** [comment, marked as answer]
Default project is a special project that is saved within the browser itself. Currently, there is no direct feature to export the project, but will add it soon.

To save the project to a local folder:
1. goto `file> open folder` (or `open folder icon in the new project window) and open a folder in your computer. 
![image](https://user-images.githubusercontent.com/5336369/205550425-5789a5d8-cff1-49d8-b99e-26516b185dc9.png)
2. Switch to default project 
![image](https://user-images.githubusercontent.com/5336369/205550342-e7658202-cd34-419a-bb9f-a1326342ad8f.png)
3. copy the files in your project, switch to the other project folder, and paste there. The files should now appear in windows explorer.


#### Please see below screen recording on how to copy:
![Peek 2022-12-05 09-58](https://user-images.githubusercontent.com/5336369/205549896-a02a0249-c09e-4523-8d48-93edc087a430.gif)
